### ISSUE TITLE
Release for using maskPlaceholder instead of maskChar

### ISSUE THREAD

**Issue by chupee120** [sanniassin/react-input-mask#212]
Hello,

Is there a new release planned for using maskPlaceholder instead of maskChar ?

Thanks,

**Comment by lelukas** [comment]
Yeah I spent like 20 minutes trying to figure out why maskPlaceholder wasn't working

**Comment by ahmedam55** [comment]
Me too. Please release it!

**Comment by rafwell** [comment]
Try install with @next and the maskPlaceholder will work. 

yarn add react-input-mask@next
